src/mocha.opts

```diff
@@ -1,3 +1,3 @@
 --require ./src/setup
 --ui bdd
---timeout 5000
+--timeout 900000
```

Comment:
I think this is way too much (250 hours). Maybe something like 15 minutes should do the trick?

Reply:
As I understood, timeout it's in milliseconds. And 900000 is 15 minutes.

Reply:
Ahhh, my bad :P Looks good then

Reply:
No worries :P